app/styles/components/system-overlay/_system-metadata.scss

```diff
@@ -1,4 +1,23 @@
 .system-metadata {
   .metadata-value { word-wrap: break-word; }
   .alert-danger { @include rem('margin-top', $global-padding); }
-}
\ No newline at end of file
+
+  .editable {
+
+    .edit-toggle {
+      display: none;
+      font-size: 18px;
+      margin-left: 8px;
```

Comment:
Let's use the `rem` formula which will convert px to rems instead of using regular px values. 
9: `@include rem('font-size', 18px);`
10: `@include rem('margin-left', 8px);`